application/build.xml

```diff
@@ -363,7 +363,6 @@
 		</zip>
 		<taskdef name="py-run" onerror="report" classname="org.pyant.tasks.PythonRunTask" classpath="${dir.tools}/pyAntTasks-1.3.jar"/>
 		<py-run script="${dir.postjad.new}/utilities/submit_build.py"/>
-		<delete dir="${build.zip.dir}"/>

```

Comment:
Do we need to move this to before the <zip> so that workspaces which already have the file around won't be overwritten? Did you try to run it twice in a row locally and make sure there weren't any issues?